index.js

```diff
@@ -0,0 +1,88 @@
+/* Set Date to Current Date */
```

Comment:
All of this code is global, which isn't good practice. You could wrap it in an IFFE function to contain it to this one script/page.
```
(function() {
  // All of the rest of the JavaScript…
})();
```

Reply:
This is homework for @ShooX512